ddl/job_table.go

```diff
@@ -266,22 +266,21 @@ func (d *ddl) startDispatchLoop() {
 		logutil.BgLogger().Fatal("dispatch loop get cluster state failed, it should not happen, please try restart TiDB", zap.Error(err))
 	}
 	defer ticker.Stop()
-	isFirst := true
 	isOnce := false
 	for {
 		if isChanClosed(d.ctx.Done()) {
 			return
 		}
-		if err := d.needCheckClusterState(isFirst || isOnce); err != nil {
-			continue
-		}
-		isFirst = false
 		if !d.isOwner() {
 			isOnce = true
 			d.once.Store(true)
 			time.Sleep(dispatchLoopWaitingDuration)
 			continue
 		}
+		if err := d.needCheckClusterState(isOnce); err != nil {
```

Comment:
Why move to here? Don't want to `checkClusterState` in first time? And maybe we could remove line 301.

Reply:
`checkClusterState` is first time in `serverStateSyncer.Init`.